torch/csrc/api/include/torch/nn/modules/conv.h

```diff
@@ -224,6 +224,28 @@ class ConvTransposeNdImpl : public ConvNdImpl<D, Derived> {
 /// Applies the ConvTranspose1d function.
 /// See https://pytorch.org/docs/master/nn.html#torch.nn.ConvTranspose1d to
 /// learn about the exact behavior of this module.
+///
+/// NOTE: `ConvTranspose1d` currently cannot be used in a `Sequential` module,
+/// because `Sequential` module doesn't support modules with forward method that can take
```

Comment:
nit: "can take" is strange wording -- it *does* take it right?  I.e. this should say "forward method that takes optional arguments"